packages/xod-client-electron/src/app/utils.js

```diff
@@ -12,3 +15,42 @@ export const errorToPlainObject = R.when(
     R.identity,
   ])
 );
+
+// :: String -> Maybe Path
+const maybePath = R.ifElse(
+  path.isAbsolute.bind(path),
```

Comment:
The `maybePath` name is misguiding. Why we ever need a voodoo dance with `isAbsolute`? I’m sure there is a reason, but I don’t understand which one. A comment will help.